.pre-commit-config.yaml

```diff
@@ -71,3 +85,39 @@ repos:
     rev: 6.3.0
     hooks:
       - id: pydocstyle
+
+  - repo: https://github.com/Lucas-C/pre-commit-hooks
+    rev: v1.5.4
+    hooks:
+      - id: forbid-crlf
+      - id: remove-crlf
+      - id: insert-license
+        files: '.*\.(py|pyi|yaml|yml|sh|helmignore|dockerignore|gitignore)$'
+        args:
+          - --license-filepath
+          - license_header.txt
+          - --comment-style
+          - "#"
+          - --use-current-year
+          - --allow-past-years
+          - --skip-license-insertion-comment=This file is maintained by velocitas CLI, do not modify manually.
+      - id: insert-license
+        files: '.*\.(cpp|c|cc|h)$'
+        args:
+          - --license-filepath
+          - license_header.txt
+          - --comment-style
+          - /**| *| */
+          - --use-current-year
+          - --allow-past-years
+          - --skip-license-insertion-comment=This file is maintained by velocitas CLI, do not modify manually.
+      - id: insert-license
+        files: "Dockerfile.*"
+        args:
+          - --license-filepath
+          - license_header.txt
+          - --comment-style
+          - "#"
+          - --use-current-year
+          - --allow-past-years
+          - --skip-license-insertion-comment=This file is maintained by velocitas CLI, do not modify manually.
```

Comment:
Not sure. I am using this pre-commit-config to perform a change of license header. If we separate these changes - it will need tone of manual work.

Reply:
Ok, then we should also note that in the commit message.
Question: Is it only adding the current year in the changed files or in all ones?

Reply:
Depends on context... If pre-commit integrated in git hook - it will perform operation only on changed files. With command `pre-commit run -a` it will touch all files.